server/controllers/ssotRetrievalController.js

```diff
@@ -184,15 +164,11 @@ exports.overwriteRobot = async (req, res) => {
 
     const ssotData = await mongoose
       .model('SSoT')
-      .findByIdAndUpdate(
-        updatedSsot['_id'],
-        updatedSsot,
-        {
-          new: true,
-          useFindAndModify: false,
-          upsert: true
-        }
-      )
+      .findByIdAndUpdate(updatedSsot['_id'], updatedSsot, {
```

Comment:
Please try if it works with dot notation or disable listing here

Reply:
✅ Is now in dot notation